src/appleseed.studio/mainwindow/mainwindow.cpp

```diff
@@ -1132,7 +1132,7 @@ void MainWindow::slot_open_cornellbox_builtin_project()
     if (!can_close_project())
         return;
 
-    const bool successful = m_project_manager.load_builtin_project("cornell_box");
+    APPLESEED_UNUSED const bool successful = m_project_manager.load_builtin_project("cornell_box");
```

Comment:
I find this quite surprising. Doesn't the compiler complain in debug? Are we sure this is going to work with other compilers?